server-access-nss/README.md

```diff
@@ -1,21 +1,23 @@
 # libnss_flantauth
 
-A library to store users in an sqlite db.
+A library to store users in a sqlite db.
```

Comment:
Not a big deal. It is the same as "a http" or "an http". Even documentation has "an sqlite": https://www.sqlite.org/quickstart.html =)

Reply:
Ok ok, it was underscored by some tool :)

Reply:
It should be “an” if your pronounce is “Ess-Cue-El-Ight”, but “a” in case of “sequel”. In this [video](https://youtu.be/Jib2AmRb_rk) from the SQLite developer you can find first pronunciation option with the first vowel sound. So I guess we should use “an” article here.